docs-src/references/dynamic-configuration.md

```diff
@@ -61,65 +61,47 @@ testGetMapPropertyKey:
 
 ### Constraints
 
-Some dynamic configuration keys are set globally, and others can be customized on a Namespace or Task Queue level.
+You can define constraints on some dynamic configuration keys to set specific values that apply on a Namespace or Task Queue level.
+Not defining constraints on a dynamic configuration key sets the values globally across the Cluster.
 
 - To set global values for the configuration key with no constraints, use:
 
   ```yaml
-  keyName:
-    - value: "my-value"
+  history.rps: # Rate limit on the number of requests made per second to each History service host.
+    - value: 5000 # The default value for this is 3000.
   ```
 
 - For keys that can be customized at Namespace level, you can specify multiple values for different Namespaces in addition to one default value that applies globally to all Namespaces.
   To set values at a Namespace level, use `namespace` (String) as shown in the following example.
 
   ```yaml
-  keyName:
+  frontend.rps: # Rate limit on the number of requests sent to the Frontend host.
```

Comment:
this key is global, it can't be customized per namespace. maybe use `frontend.globalNamespaceRPS`

Reply:
Thanks for clarifying this; wasn't sure if I should use the global namespace one. updated!

Reply:
this wasn't updated?

Reply:
this still wasn't changed. `frontend.rps` is not a key that can be customized by namespace

Reply:
I keep missing this update! Sorry about that. Can I use `frontend.persistenceNamespaceMaxQPS` instead (because I've already used the `frontend.globalNamespaceRPS` in the prev example.)

Reply:
Yes, that one accepts namespace constraints

Reply:
updated the example to use:

  ```yaml
  frontend.persistenceNamespaceMaxQPS: # Rate limit on the number of queries the Frontend sends to the Persistence store.
    - constraints: {} # Sets default value that applies to all Namespaces
      value: 2000 # The default value for this key is 0.
    - constraints: {namespace: "namespace1"} # Sets limit on number of queries that can be sent from "namespace1" Namespace to the Persistence store.
      value: 4000
    - constraints: {namespace: "namespace2"}
      value: 1000
  ```